packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -562,13 +562,11 @@ class FloatingLabelAlignment {
   }
 
   static String _stringify(double x) {
-    if (x == -1.0) {
-      return 'FloatingLabelAlignment.start';
-    }
-    if (x == 0.0) {
-      return 'FloatingLabelAlignment.center';
-    }
-    return 'FloatingLabelAlignment(x: ${x.toStringAsFixed(1)})';
+    return switch (x) {
+     -1.0 => 'FloatingLabelAlignment.start',
```

Comment:
nit: should have 2-character indentation:)
```suggestion
      -1.0 => 'FloatingLabelAlignment.start',
```

Reply:
It looks like auto-submit bot kicked in right when you submitted the review 😒

I thought it looked nice with all the numbers aligned, but you're right—in the future I'll be better about following this convention!

Reply:
Ah no worries at all! Thanks:)!